senseact/envs/ur/reacher_env.py

```diff
@@ -592,6 +603,7 @@ def _handle_bounds_speedj(self):
                              np.all(self._qt_[-1, self._joint_indices] <= self._angles_high)
         if inside_bound:
             self.return_point = None
+            self.escaped_the_box = False
```

Comment:
I'm not seeing anywhere this variable actually being used. Could we just remove it?